cluster/centos/config-default.sh

```diff
@@ -117,7 +120,7 @@ export FLANNEL_NET=${FLANNEL_NET:-"172.16.0.0/16"}
 
 # Admission Controllers to invoke prior to persisting objects in cluster
 # If we included ResourceQuota, we should keep it at the end of the list to prevent incrementing quota usage prematurely.
-export ADMISSION_CONTROL=NamespaceLifecycle,LimitRanger,ServiceAccount,ResourceQuota,DefaultTolerationSeconds
+export ADMISSION_CONTROL=${ADMISSION_CONTROL:-"NamespaceLifecycle,LimitRanger,ServiceAccount,ResourceQuota,DefaultTolerationSeconds"}
```

Comment:
Support defined `ADMISSION_CONTROL` outside by user, `DefaultTolerationSeconds` is not exists before [cd427fa4](https://github.com/kubernetes/kubernetes/commit/cd427fa4bece7d0add1be70d12e347e60b637f50).